servers/visual/visual_server_scene.cpp

```diff
@@ -1522,6 +1522,13 @@ void VisualServerScene::instance_geometry_set_material_override(RID p_instance,
 }
 
 void VisualServerScene::instance_geometry_set_draw_range(RID p_instance, float p_min, float p_max, float p_min_margin, float p_max_margin) {
```

Comment:
The documentation should be updated (it still says "Not implemented in Godot 3.x.").